config/webpack.config.js

```diff
@@ -25,7 +25,7 @@ const PFGenerator = asGenerator((item, ...rest) => {
   return defaultTuples;
 });
 
-const publicPath = process.env.BETA === 'true' ? '/beta/apps/chrome/js/' : '/apps/chrome/js/';
+const publicPath = process.env.BETA === 'true' ? '/apps/chrome/js/' : '/apps/chrome/js/';
```

Comment:
What's the point in the ternary operatory provided that the result is always '/apps/chrome/js/'?

Reply:
None. I'll fix it.